go-controller/pkg/node/node.go

```diff
@@ -495,6 +498,15 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	}
 
 	if config.HybridOverlay.Enabled {
+		var libovsdbNBClient libovsdbclient.Client
+		var err error
+
+		stopChan := make(chan struct{})
+		defer close(stopChan)
```

Comment:
Shouldn't this be closed in the go func below?

Reply:
Yep